_episodes/09-cmorization.md

```diff
@@ -0,0 +1,1053 @@
+
+```
+DEBUG   Retrieving OBS6 configuration
+DEBUG   Skipping non-existent /home/peter/default_inputpath/Tier3/FLUXCOM
+DEBUG   Looking for files matching ['OBS6_FLUXCOM_reanaly_ANN-v1_Lmon_gppStderr[_.]*nc'] in []
+ERROR   No input files found for variable {'variable_group': 'gppStderr' ...
+...
+ERROR   Missing data for preprocessor FLUXCOM/gppStderr
+...
+ERROR   Program terminated abnormally, see stack trace below for more information:
+...
+```
+{: .error}
+
+So the output tells us that it cannot find the data, and also where it has been
+looking. Let's try to understand what's going on.
+
+## Location, location, location
+
+The error messages above tell you that ESMValTool cannot find the data. That
+makes total sense, because we have not yet created our CMORized copy of the
+data. Our data is located in the `RAWOBS` folder, but ESMValTool is looking in
+`OBS6`. So the first thing our CMORizer will need to do is copy the data over
+from one folder to the other. To do end, we need to tell ESMValTool where our
+data may be found.
+
+> ## Set the correct paths in your user configuration file:
+>
+> This information is set in `config-user.yml`. Modify your
+> configuration file so that it has the correct paths
+>
+> ```yaml
+> rootpath:
+>   OBS6: /path/to/my/obs6/data
+>   RAWOBS: /path/to/my/rawobs/data
+> ```
+{: .challenge}
+
+> ## RAWOBS, OBS, OBS6!?
+>
+> ESMValTool uses project IDs to find the data on your hard drive, and also to
+> find more information about the data. The `RAWOBS` and `OBS` projects were
+> created for external data before and after CMORization, respectively. These
+> names can be misleading, though, since not all external datasets are
+> observations.
+>
+> Then, in going from CMIP5 to CMIP6, the CMOR standards changed a bit. Some
+> variables are named differently in CMIP5 and CMIP6. This posed a dilemma:
+> should CMORization reformat to the CMIP5 or CMIP6 definition? To solve this,
+> the `OBS6` project was created. So data in the `OBS6` follow the CMIP6
+> standards.
+>
+{: .callout}
+
+Looking closer at the path that ESMValTool told us it went looking for the data,
+we see that it has a very specific structure:
+
+`OBS6_FLUXCOM_reanaly_ANN-v1_Lmon_gppStderr[_.]*nc`
+
+Note that all components in the filename correspond to one of the dataset keys
+defined in our test recipe. By replacing them with the original tags, we can
+see the general structure for the OBS6 data:
+
+`[project]_[dataset]_[type]_[version]_[mip]_[short_name]_*.nc`
+
+This is the Data Reference Syntax (DRS) for OBS6 data. Each project has its own
+DRS, and some project IDs even have multiple different options. These settings
+can be found in the file
+[config-developer.yml](https://github.com/ESMValGroup/ESMValCore/blob/master/esmvalcore/config-developer.yml).
+Luckily, we don't need to worry about that. This file name will be automatically
+correctly created when we run our CMORizer script.
```

Comment:
In this part, I am introduced to an example of DRS and then to the definition of DRS. Can we reverse the order i.e. first explain what a DRS is and then show me an example and then a reference to config-developer? 

Another point, in episode **configuration**, there is a section **Directory structure for the data from different projects** that explains DRS. Can we point users to that section?

Reply:
I removed most of this.